helpdesk_mgmt_project/controllers/main.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+import odoo.http as http
+
+from odoo.addons.helpdesk_mgmt.controllers.main import HelpdeskTicketController
+
+_logger = logging.getLogger(__name__)
+
+
+class CustomHelpdeskTicketController(HelpdeskTicketController):
+    def _prepare_submit_ticket_vals(self, **kw):
+        vals = super(CustomHelpdeskTicketController, self)._prepare_submit_ticket_vals(
+            **kw
+        )
+        team = (
+            http.request.env["helpdesk.ticket.team"].sudo().browse(vals.get("team_id"))
+        )
+        if team.default_project_id and not vals.get("project_id"):
```

Comment:
@LoisRForgeFlow , it will raise `AttributeError: 'helpdesk.ticket.team' object has no attribute 'default_project_id'` because `default_project_id` is defined in `helpdesk_mgmt_timesheet`.
This `main.py` should be moved to the `helpdesk_mgmt_timesheet` module that depends on `helpdesk_mgmt_project`

Reply:
@ypapouin You are actually right... thanks for catching it. Please open a PR fixing it and ping me, I'll be glad to review it.